libs/designsystem/src/lib/components/action-sheet/action-sheet.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../scss/utils';
+
+:host {
+  position: relative;
+  display: inline-block;
+  outline: none;
+}
+
+:host(.is-open) {
```

Comment:
Consider also adding a `.is-opening` class to allow for animating the transition from closed => open (see dropdown for an example)

Reply:
See previous comment